/**:
  ros__parameters:
    use_inspva_heading: false
    bestvel_heading_update_velocity_thres : 3.0

    show_debug_info : false
    predict_frequency : 100.0
    tf_rate : 10.0
    enable_yaw_bias_estimation : false
    extend_state_step : 20

    pose_additional_delay : 0.0
    pose_measure_uncertainty_time : 0.05
    pose_rate : 20.0
    pose_gate_dist : 3.0 # this value is margin. mahalanobisGate is updated based on vehicle speed.
    pose_stddev_x : 0.050 # this value is replaced by novatel insstdev
    pose_stddev_y : 0.050 # this value is replaced by novatel insstdev
    pose_stddev_yaw : 0.005 # this value is replaced by novatel insstdev
    use_pose_with_covariance : false

    twist_additional_delay : 0.0
    twist_rate : 100.0
    twist_gate_dist : 20.0
    twist_stddev_vx : 0.2
    twist_stddev_wz : 0.03
    use_twist_with_covariance : false

    proc_stddev_yaw_c : 0.005 #DO NOT TOUCH
    proc_stddev_yaw_bias_c : 0.001 #DO NOT TOUCH
    proc_stddev_vx_c : 5.0 #10.15 # 0.15 #5.0
    proc_stddev_wz_c : 1.05 #1.05

    use_bestvel_for_speed : false

    #To compare with INS result
    ins_top_to_gnss_bias_x : -0.35
    ins_top_to_gnss_bias_y : -0.6
    

    # new center transform bias
    top_to_bottom_bias_x : 0.0
    top_to_bottom_bias_y : 0.52        
    bottom_to_top_bias_x : -1.11
    bottom_to_top_bias_y :  0.0

    # reference gps error information
    use_reference_information : false
